tests/system/system_test.go

```diff
@@ -22,15 +23,22 @@ func NewSystemTest(t *testing.T, dir string, vars map[string]string) *SystemTest
 	vars["endpoint"] = os.Getenv(config.ENVVAR_ENDPOINT)
 	vars["token"] = os.Getenv(config.ENVVAR_TOKEN)
 
+	fmt.Println("[DEBUG] endpoint", vars["endpoint"])
```

Comment:
always use `log` package to log! Also, this should probably be cleaned up a bit using `Printf`

Reply:
✅ using testing.T.Logf

Reply:
You may want to consider using `log.Printf("[DEBUG] ...")` over `t.Logf()`, because they have slightly different behaviors:
* Any `log.Printf("[DEBUG] ...)` statements print if the user runs `go test --debug`. These statements print immediately. 
* Any `t.Logf` statements print if the user runs `go test -v`. These statements print only after the test has completed or failed.